test/browser_tests/conf.js

```diff
@@ -2,27 +2,80 @@
 
 var env = require( './environment.js' );
 
-exports.config = {
+// Set browsers to test.
+// See https://docs.saucelabs.com/reference/platforms-configurator/
+// for configuration values for version and platform.
+
+// Essential browsers for running locally.
+var _essential = {
   multiCapabilities: [
     {
       browserName: 'chrome',
```

Comment:
Depends which should be launched when testing locally. The thinking here was instead of running multiple browsers locally, we run one as a sanity check and then dump the whole spectrum of browsers onto Sauce Labs, which could then be run in a separate Terminal tab if desired and not be launching a lot of browsers locally. My perception is Chrome manages resources and opens quicker than FF, leading to quicker test execution, but maybe I've been brainwashed by Google.